ppr-api/test_data/postgres_data_files/test0001.sql

```diff
@@ -37,7 +37,7 @@ INSERT INTO parties(id, party_type, registration_id, financing_id, registration_
                   middle_initial, last_name, business_name, birth_date, address_id,
                   business_srch_key, bus_name_base, bus_name_key_char1)
     VALUES(200000002, 'DB', 200000000, 200000000, null, null, null, null, null, 'TEST BUS 2 DEBTOR',
-           null, 200000002, searchkey_business_name('TEST BUS 2 DEBTOR'), business_name_strip_designation('TEST BUS 2 DEBTOR'), 'T')

```

Comment:
why the change here? I think the `business_name_strip_designation` is how the code inserts it in so it should be done here as well right?